lynx/common/scene.py

```diff
@@ -10,28 +11,42 @@
 @dataclass
 class Scene(Serializable):
     entities: List[Entity] = field(default_factory=list)
-    _object_position_map: Dict[Vector, Object] = field(default_factory=dict)
+    _square_position_map: Dict[Vector, Square] = field(default_factory=dict)
     _object_id_map: Dict[int, Object] = field(default_factory=dict)
 
+    def _get_square(self, position: Vector) -> Square:
+        # This method guarantees that the square is correct
+        if square := self._square_position_map.get(position):
+            return square
+        else:
```

Comment:
There we can remove `else` because if condition (square := self._square_position_map.get(position)) is True we return square

Reply:
It's a little less clear in my opinion, but sure - it's generally better.

Reply:
I don't know what others think but I like use `early return`